src/shared/sam/localLambdaRunner.ts

```diff
@@ -86,43 +86,24 @@ const ATTACH_DEBUGGER_RETRY_DELAY_MILLIS: number = 1000
 let samStartApi: Promise<boolean>
 let samStartApiProc: ChildProcess | undefined
 
-export async function makeInputTemplate(config: SamLaunchRequestArgs): Promise<string> {
+export async function makeInputTemplate(
+    config: SamLaunchRequestArgs & { invokeTarget: { target: 'code' } }
+): Promise<string> {
     let newTemplate: SamTemplateGenerator
-    let inputTemplatePath: string
+    const inputTemplatePath: string = path.join(config.baseBuildDir!, 'app___vsctk___template.yaml')
     const resourceName = makeResourceName(config)
 
-    // use existing template to create a temporary template
-    if (['api', 'template'].includes(config.invokeTarget.target)) {
-        const template = getTemplate(config.workspaceFolder, config)
-        const templateResource = getTemplateResource(config.workspaceFolder, config)
+    // code type - generate ephemeral SAM template
+    newTemplate = new SamTemplateGenerator()
+        .withFunctionHandler(config.invokeTarget.lambdaHandler)
+        .withResourceName(resourceName)
+        .withRuntime(config.lambda!.runtime!)
+        .withCodeUri(pathutil.normalize(config.invokeTarget.projectRoot))
```

Comment:
mmmm... we are still generating stuff, but not testing it anymore? Or at least, it looks like a lot of the test cases were removed.

Reply:
We need to generate something for the 'code' target case. That's still tested, though most of the test logic is redundant.